pkg/client/route/client.go

```diff
@@ -54,7 +53,7 @@ func New(ctx provider) (*Client, error) {
 		return nil, errors.New("cast service to route service failed")
 	}
 
-	connectionLookup, err := connectionstore.NewConnectionLookup(ctx)
+	connectionLookup, err := connection.NewConnectionLookup(ctx)
```

Comment:
This probably could have been connection.NewLookup

Reply:
done